src/main/java/Main.java

```diff
@@ -226,6 +227,8 @@ private static void _main(String[] args) throws Exception {
             arguments.add("--webroot="+new File(describedHomeDir.file,"war"));
         }
 
+        deleteWinstoneTempContents(extractedFilesFolder,"winstone.*\\.jar", "jna-.*");
```

Comment:
Deleting `jna-*` is unsafe because these files are produced by JNA and could be in use by unrelated processes.

Not exactly clear that it is safe to delete even `winstone*.jar`. Would break the case that two Jenkins services are being run on the same machine on different ports. Not a likely scenario, but AFAIK would have worked except for this patch.

Reply:
we could add a property to disable the behavior and a warning to avoid it could happen